server/test/server.spec.ts

```diff
@@ -0,0 +1,27 @@
+// 
+// Note: This example test is leveraging the Mocha test framework.
+// Please refer to their documentation on https://mochajs.org/ for help.
+//
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as path from 'path';
+import { error } from 'util';
+import * as server from '../src/server';
+
+// Defines a Mocha test suite to group tests of similar kind together
+describe("Server Tests", () => {
+
+
+	// test validation cto
+	it("validateCtoModelFile", () => {
```

Comment:
All mocha tests should follow a specific format if using a 'describe' framework, as shown here: https://mochajs.org/

This is so that the actual test intention is clear to a third party